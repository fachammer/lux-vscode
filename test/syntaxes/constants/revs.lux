.123,456,789
.0
.123,,234,,,25,34,46,6,8
