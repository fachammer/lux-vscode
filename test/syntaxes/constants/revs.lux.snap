>.123,456,789
#^^^^^^^^^^^^ source.lux constant.numeric.rev.lux
>.0
#^^ source.lux constant.numeric.rev.lux
>.123,,234,,,25,34,46,6,8
#^^^^^^^^^^^^^^^^^^^^^^^^ source.lux constant.numeric.rev.lux
>